doc/source/user_guide/tutorial_segmentation.rst

```diff
@@ -17,21 +17,21 @@ thresholding segmentation is not sufficient.
 
 ::
 
-    >>> import numpy as np
     >>> from skimage import data
+    >>> from skimage.exposure import histogram
     >>> coins = data.coins()
-    >>> histo = np.histogram(coins, bins=np.arange(0, 256))
+    >>> hist = histogram(coins)
```

Comment:
why not ``hist, hist_centers`` as in the other examples? I like this syntax.

Reply:
Good point. I fixed it.